tests/admin.test.ts

```diff
@@ -71,15 +71,16 @@ describe('retroactive attendance submission', () => {
 
     await adminController.submitAttendanceForUsers(
       { users: [user.email], event: event.uuid },
-      proxyUser,
+      admin,
     );
 
-    const userResponse = await userController.getUser({ uuid: user.uuid }, proxyUser);
-    const attendanceResponse = await attendanceController.getAttendancesForCurrentUser(user);
-    const activityResponse = await userController.getCurrentUserActivityStream(user);
-
+    const userResponse = await userController.getUser({ uuid: user.uuid }, admin);
     expect(userResponse.user.points).toEqual(user.points);
+
+    const attendanceResponse = await attendanceController.getAttendancesForCurrentUser(user);
     expect(attendanceResponse.attendances).toHaveLength(1);
+
+    const activityResponse = await userController.getCurrentUserActivityStream(user);
```

Comment:
Same here, let's verify the activity objects.

Reply:
Resolved in 6688e90.